ChangeLog.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
```

Comment:
LGTM 🚀 

Let's just add this under `v0.3.9` and add that to `__version__` as well 👍

Reply:
Thanks 👍